Coproc fix chaos

VadimPlh opened the pull request:
## Cover letter

Describe in plain language the motivation (bug, feature, etc.) behind the change in this PR and how the included commits address it.

Fixes: #NNN, #NNN, ...

## Release notes

If the PR changes the user experience, write a short summary of the changes. See the [CONTRIBUTING](https://github.com/vectorizedio/redpanda/blob/dev/CONTRIBUTING.md) guidelines for details.

Release note: [1-2 sentences of what this PR changes]

graphcareful commented on tests/rptest/wasm/native_kafka_producer.py:
```diff
@@ -33,6 +36,9 @@ def task_name(self):
 
     def _run(self):
+        producer = KafkaCliTools(self._redpanda)
+        producer.produce(self._topic, self._num_records, self._records_size)
```
Technically this class shouldn't be called `NativeKafkaProducer` if your going to use `kafkaCliTools` as the implementation. `native` means it uses the `kafka-python` official API under the hood

VadimPlh committed:
ducktape wasm: delete WasmMeshFailureRecoveryTest

This test is broken, because wasm_xfactor must be 3
And this method is not used in wasm_tests, so
size input topic is not same wich output topic

VadimPlh committed:
wasm ducktape: use Cli tools in producer for wasm tests

VadimPlh committed:
ducktape wasm: switch on wasm tests in CI

VadimPlh committed:
ducktape wasm: Add zstandard to test dependencies

VadimPlh committed:
ducktape wasm:  increase waiting timeout for wasm tests
